docs/farming-&-staking/staking/operators/tips-operator.mdx

```diff
@@ -13,34 +13,20 @@ import CodeBlock from '@theme/CodeBlock';
 import Link from '@docusaurus/Link';
 import styles from '@site/src/pages/index.module.css';
 
-### Create operator key (alternative, less secure way):
-An operator needs a key pair to participate in bundle production.
```

Comment:
This is still true?

Reply:
No this is no longer true, ut's been deleted.

Reply:
I mean the deleted statement "An operator needs a key pair to participate in bundle production." is still true but we've removed it?

Reply:
:100: needs a key pair

Reply:
Sure, the operator of course needs a key pair. It's just the phrase itself was a preamble of a keypair generation, which is no longer relevant because the way the keys are generated is completely new. Now there's no preferred, or backup method of generating the keypair, so the whole section in the "tips-operator" is dropped, alongside with the sentence.

Reply:
I moved the phrase to `register-operator.md` under the **Create operator key** section.